R/deployments-find.R

```diff
@@ -37,9 +44,14 @@ disambiguateDeployments <- function(appDeployments, error_call = caller_env()) {
   }
 
   apps <- paste0(
-    appDeployments$name, " ",
-    "(", accountLabel(appDeployments$account, appDeployments$server), "): ",
-    "{.url ", appDeployments$url, "}"
+    appDeployments$name,
+    " ",
+    "(",
+    accountLabel(appDeployments$account, appDeployments$server),
+    "): ",
+    "{.url ",
+    appDeployments$url,
+    "}"
```

Comment:
This one seems worse. I'd put the `#fmt: skip` comment in myself, but it should probably go above line 46.

Reply:
This one, I've rewritten to use `sprintf`. It's more readable than the original or reformatted form.